src/ui/src/builder/BuilderBlueprintState.vue

```diff
@@ -42,16 +43,18 @@ const result = computed(() => {
 	<div class="BuilderBlueprintState">
 		<slot v-if="!result" name="unknown"></slot>
 		<slot v-else-if="result === 'success'" name="success">
-			<span
-				class="BuilderBlueprintState__status BuilderBlueprintState__status--success material-symbols-outlined"
-				>check</span
+			<div
+				class="BuilderBlueprintState__status BuilderBlueprintState__status--success"
 			>
+				<WdsIcon name="check" />
+			</div>
```

Comment:
Do we need an extra `<div>`? Can't it be 
```suggestion
			<WdsIcon
				class="BuilderBlueprintState__status BuilderBlueprintState__status--success"
				name="check"
			/>
```

It applies to others.

Reply:
In that particular case we do need the extra `<div>` as it acts as a styled circle with background and fixed size, and the icon is centered inside it. If we removed the wrapper, the SVG itself would stretch to fill the space (unlike icon fonts), which would break the intended layout and styling. This setup preserves the original look we had with icon fonts.

Here is a visual example. The icon on the left has no wrapper, and the one on the right does have the wrapper:
<img width="325" height="83" alt="Screenshot 2025-07-14 at 12 44 04" src="https://github.com/user-attachments/assets/3723bed1-d5cc-474d-b248-8c92410fce82" />